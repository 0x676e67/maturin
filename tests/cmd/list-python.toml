bin.name = "maturin"
args = "list-python --help"
