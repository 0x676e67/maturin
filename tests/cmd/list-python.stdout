maturin[EXE]-list-python 
Search and list the available python installations

USAGE:
    maturin[EXE] list-python [OPTIONS]

OPTIONS:
        --target <TARGET>    
    -h, --help               Print help information
